beacon-chain/state/stateV0/state_trie.go

```diff
@@ -204,6 +205,134 @@ func (b *BeaconState) HashTreeRoot(ctx context.Context) ([32]byte, error) {
 	return bytesutil.ToBytes32(b.merkleLayers[len(b.merkleLayers)-1][0]), nil
 }
 
+// ToProto returns a protobuf *v1.BeaconState representation of the state.
+func (b *BeaconState) ToProto() *v1.BeaconState {
```

Comment:
Isn't this the same as CloneInnerState()?

Reply:
Looks like it is the same. In addition `CloneInnerState` includes some additional safety measures. Thanks for noticing!

Reply:
I looked more closely at the code today and it's actually not the same. `CloneInnerState` returns a non-ethereumapis state used by our old APIs.